daprdocs/content/en/reference/components-reference/supported-bindings/mqtt.md

```diff
@@ -136,6 +136,20 @@ This component supports **output binding** with the following operations:
 
 - `create`
 
+## Set topic per-request
+
+You can specify the topic to send the message to on a per-request basis using request metadata:
```

Comment:
Q: How does this relate to the `topic` is the spec metadata. Does this override that topic for each request and what happens if I set both and they are different? Would be good to clarify this here.

Reply:
It overrides it.